src/api/verify.ts

```diff
@@ -114,31 +114,23 @@ class VerifyAPI {
     return trackRes;
   }
 
-  static async startTrackingVerified(params: RadarStartTrackingVerifiedParams) {
-    const doTrackVerified = async () => {
-      let trackRes;
-      try {
-        trackRes = await this.trackVerified(params);
-      } catch (err: any) {
-        Logger.error(`trackVerified error: ${err.message}`);
-      }
-      
+  static startTrackingVerified(params: RadarStartTrackingVerifiedParams) {
+    const scheduleNextIntervalWithLastToken = async () => {
       const { interval } = params;
 
-      let expiresIn = 0;
       let minInterval = interval;
 
-      if (trackRes) {
-        expiresIn = (trackRes.expiresIn || expiresIn);
+      if (lastToken) {
+        const lastTokenElapsed = (performance.now() - lastTokenNow) / 1000;
+
+        const expiresIn = (lastToken.expiresIn || 0);
 
         // if expiresIn is shorter than interval, override interval
-        minInterval = Math.min(expiresIn, interval);
+        // re-request early to maximize the likelihood that a cached token is available
+        minInterval = Math.min(expiresIn - lastTokenElapsed, interval);
       }
 
-      // re-request early to maximize the likelihood that a cached token is available
-      if (minInterval > 20) {
-        minInterval = minInterval - 10;
-      }
+      minInterval = minInterval - 10;
```

Comment:
The one thing that is hard to follow in this code is what things are seconds, and what are milliseconds. I wonder if we should rename this to:
```
minIntervalSeconds
```
Just to make it more clear (as well as other things that might be in seconds / ms)

Reply:
Also, is there a reason we refer to it as `lastToken` instead of `lastResponse`?

Reply:
It's fair feedback. In this case I was just trying to mirror the mobile SDK code + var names as closely as possible to make it easy to compare across platforms